openapi/components/schemas/ExternalServiceSettings.yaml

```
@@ -0,0 +1,11 @@
+type: object
+description: Settings for external service integration.
+properties:
+  quickBooksOnline:
+    type: object
+    description: QuickBooks Online settings.
+    properties:
+      syncJournalEntries:
+        type: boolean
+        description: Specifies if journal entries should be synchronized.
```

nitpick: avoid using [should](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#general-guidance)
```suggestion
        description: Specifies if journal entries must be synchronized.
```